setup.cfg

```
@@ -37,9 +37,7 @@ python_requires = '>=3.7'
 packages = find:
 setup_requires =
     setuptools_scm
-    sphinx
-    python-docs-theme
-    pytest-runner
+    pytest_runner
 tests_require =
     pytest
     pytest-cov
```

Hi. Thank you for submitting this patch. I think `pytest_runner` should be moved into the `tests_require` section as well.
```suggestion
tests_require =
    pytest
    pytest-cov
    pytest_runner
```

---

Hello @amotl , thank you for the review! 

Moving `pytest_runner` to `tests_require` breaks the current test execution (via `python setup.py test`). 

While this should be fixed, I first only wanted to fix the issue at hand.